backend/server.py

```diff
@@ -10,6 +10,7 @@
 
 from py_utils.wait.BooleanWaitHandler import BooleanWaitHandler
 from py_utils.wait.IntWaitHandler import IntWaitHandler
+from py_utils.wait.WaitHandler import AwakeReason

```

Comment:
I recommend moving the timeout values to a separate `default_values.py` file. This would centralize all timeout settings, making it easier to modify them based on different use cases. It will improve maintainability and ensure that adjustments can be made consistently across the codebase without needing to modify individual functions. Keeping timeouts in one place ensures consistency and simplifies future changes.